src/backend/controllers/AuthController.js

```diff
@@ -0,0 +1,99 @@
+import { v4 as uuid } from "uuid";
+import { Response } from "miragejs";
+import { formatDate } from "../utils/authUtils";
+const sign = require("jwt-encode");
+/**
+ * All the routes related to Auth are present here.
+ * These are Publicly accessible routes.
+ * */
+
+/**
+ * This handler handles user signups.
+ * send POST Request at /api/auth/signup
+ * body contains {firstName, lastName, email, password}
+ * */
+
+export const signupHandler = function (schema, request) {
+  const { email, password, ...rest } = JSON.parse(request.requestBody);
+  try {
+    // check if email already exists
+    const foundUser = schema.users.findBy({ email });
```

Comment:
Is this part working ? Not sure , but I think findBy() takes to arguments and you have passed only one .

Reply:
tbh this is the backed pre generated by mockbee and not what I have written.